### PR TITLE
Vs/new logo

### PR DESCRIPTION
**Pull request by vsiri**
I added the svg and the png versions of the logo in our color yay

changed layout.jade to use the new logo png
screen shot 2016-07-23 at 4 40 03 pm

screenshot cut a bit off the top but it is centered!

@VOjha @kimberlytze

### CODE REVIEW

File: upnext/static/images/logo.svg [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	<path fill="#e7dfdd" d="M86.3,149.1v-2.2c0-0.6,0.4-1,1-1h4.1c0.6,0,1,0.4,1,1v2.2c0,0.6-0.4,1-1,1h-4.1C86.8,150.1,86.3,149.6,86.3,149.1z"/>
+</g>
+<g>
+	<path fill="#e7dfdd" d="M95.1,149.1v-2.2c0-0.6,0.3-1,0.9-1h0.3c0.6,0,1,0.4,1,1v2.2c0,0.6-0.4,1-1,1H96C95.4,150.1,95.1,149.6,95.1,149.1z"/>
+</g>
+<path fill="#e7dfdd" d="M95.9,130.9c0,0,9.3-0.2,9.2,9.6c-0.1,9.9-9.2,9.5-9.2,9.5v-4.2c0,0,5.2,1.3,5.1-4.9c-0.1-6.2-2.4-5.9-7.3-5.9
+	S95.9,130.9,95.9,130.9z"/>
+<path fill="#e7dfdd" d="M95.9,150.1c0,0,3.4,0.3,6.7-2.7c3.2-3-7.2,1.9-7.2,1.9L95.9,150.1z"/>
+<path fill="#e7dfdd" d="M96,145.9c0,0,4.5,0.2,4.9-3.5c0.4-3.7-0.2,3.5-0.2,3.5S96.2,147.6,96,145.9z"/>
+</svg>
```
Comment (VOjha) [inline upnext/static/images/logo.svg:58]: o god

**Comment by kimberlytze** [comment]
d o p e

**Comment by VOjha** [comment]
sick logo bruh :ship:

**Comment by vsiri** [comment]
thanks michael lee

**Pull request closed without merging**